pkg/auth/authorizer/abac/abac.go

```diff
@@ -215,7 +290,12 @@ func resourceMatches(p api.Policy, a authorizer.Attributes) bool {
 }
 
 // Authorizer implements authorizer.Authorize
-func (pl policyList) Authorize(a authorizer.Attributes) error {
+func (apolicy *ABACPolicy) Authorize(a authorizer.Attributes) error {
+	pl, ok := apolicy.Auth.Load().(policyList)
```

Comment:
this is going to hit every request... I don't know how Load() performs. Seems like we should only do this in cases where we're supporting reload (which I expect to be a vanishingly small set)